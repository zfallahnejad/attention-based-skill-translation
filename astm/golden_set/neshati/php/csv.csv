324584,csv
476,csv
1491895,csv
345031,csv
1226894,csv
3859027,csv
2394511,csv
171318,csv
285587,csv
1415724,csv
2364629,csv
20862,csv
250259,csv
