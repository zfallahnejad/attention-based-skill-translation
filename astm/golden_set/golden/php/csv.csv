324584,24,29,0.4528301886792453
1491895,17,4,0.8095238095238095
476,17,11,0.6071428571428571
1226894,12,5,0.7058823529411765
345031,12,9,0.5714285714285714
3859027,10,0,1.0
2394511,9,9,0.5
20862,7,6,0.5384615384615384
1415724,7,6,0.5384615384615384
250259,7,10,0.4117647058823529
285587,7,3,0.7
171318,7,6,0.5384615384615384
2364629,7,8,0.4666666666666667
